packages/components/src/autocomplete/index.js

```diff
@@ -472,9 +472,9 @@ function Autocomplete( {
 		}
 
 		const safeTrigger = escapeRegExp( completer.triggerPrefix );
-		const match = text.match(
-			new RegExp( `${ safeTrigger }([\u0000-\uFFFF]*)$` )
-		);
+		const match = text
+			.slice( text.lastIndexOf( completer.triggerPrefix ) )
```

Comment:
Good catch, it makes a lot of sense.

It feels like we could update the `RegExp` by matching the last occurrence, but I'm afraid it would be harder to follow who need to google like me:
https://frightanic.com/software-development/regex-match-last-occurrence/